build.sbt

```diff
@@ -3,9 +3,9 @@ import xerial.sbt.Sonatype.autoImport._
 lazy val commonSettings = Seq(
   organization := "com.github.seratch",
   name := "awscala",
-  version := "0.8.3",
-  scalaVersion := "2.12.8",
-  crossScalaVersions := Seq("2.12.8", "2.11.12", "2.10.7"),
+  version := "0.8.4-SNAPSHOT",
+  scalaVersion := "2.13.1",
+  crossScalaVersions := Seq("2.12.10", "2.13.1"),
```

Comment:
It should be acceptable to drop scala 2.10 and 2.11 support. The benefit by this would be that we can simplify the code for cross-builds.